posthog/session_recordings/session_recording_api.py

```diff
@@ -248,6 +247,20 @@ def destroy(self, request: request.Request, *args: Any, **kwargs: Any) -> Respon
 
         return Response({"success": True}, status=204)
 
+    @action(methods=["POST"], detail=True)
+    def persist(self, request: request.Request, *args: Any, **kwargs: Any) -> Response:
+        recording = self.get_object()
+
+        if not settings.EE_AVAILABLE:
+            raise exceptions.ValidationError("LTS persistence is only available in the full version of PostHog")
+
+        # Indicates it is not yet persisted
+        # "Persistence" is simply saving a record in the DB currently - the actual save to S3 is done on a worker
+        if recording.storage == "object_storage":
```

Comment:
Dang, this was only a debug flag previously so I've stripped it out in the yeet PR 😢